size-auditor/size-auditor.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Size auditor Configuration",
+  "description": "Configuration file for the Size Auditor tool",
+  "type": "object",
+  "properties": {
+    "sizeTolerance": {
+      "description": "The tolerance limit above which Size Auditor would fail a pull request",
+      "type": "number"
+    },
+    "trackingEntity": {
+      "description": "The entity that is audited for size regressions by Size Auditor",
+      "type": "string",
+	  "pattern": "/a@(Asset|Chunk|Assembly)\b/"
+    },
+    "devopsDropFolderName": {
+      "description": "The name of the drop location containing build artifacts",
+      "type": "string"
+    },
+	"devopsAssemblyArtifactName": {
+      "description": "The name of the artifact folder on Azure Devops that contains information related to Assembly sizes for the build",
```

Comment:
Is "Assembly" sizes an external concept? Can it be worded more generically since you support other types of assets?

Also similar comment to the one above re: path clarifications.

Reply:
Will think about this a bit more. The way I have it now, is a union of all possible configs across all repositories that use size auditor.

"Assembly" is specific to just one repo (`sp-client`), but we might have other repositories in future where we may have to track two or more artifact locations, resulting in an entry for each type of `asset` we are tracking.

Reply:
thanks for the comment!